src/sched/lthread.c

```diff
@@ -634,7 +635,11 @@ int lthread_create_primitive(
     lt->robust_list.head = &lt->robust_list.head;
 
     static unsigned long long n = 0;
-    snprintf(lt->funcname, 64, "cloned host task %llu", __atomic_fetch_add(&n, 1, __ATOMIC_SEQ_CST));
+    snprintf(
```

Comment:
Please avoid whitespace-only changes in things that have been modified in other open PRs with functional changes.

Reply:
Sorry, I auto-format, I won't use a different file saving process depending on arbitrarily made up rules. If there's a good reason to use a different style for some of the files, that must be automated, otherwise it has no chance not to end up a mess.

Reply:
@wintersteiger If you know about your auto-formatting setting then why not create a separate PR first that auto-formats everything?

Reply:
@prp did exactly that before the repo move (on my request). 

I sometimes don't notice because the bits that I edit are already formatted correctly, while some other parts of a file may not be.

Reply:
You are making non-functional changes that are going to cause merge conflicts.  This is not helpful.  You are increasing the size of the diff with non-functional changes, which increases the reviewer burden on an already-large feature.  This is not helpful.

Reply:
Yes, which is a one-time occasion, after which the problem is fixed forever. Sure, I can separate those bits into a separate PR, but that takes exactly the same amount of time it takes you to skip over it. If you want me to change it in this PR here and now, please say so clearly.

Reply:
I would like you to not raise PRs that have whitespace-only changes interleaved with non-whitespace changes.  This is particularly problematic for bits of code that have already been modified in other open PRs, because you are *guaranteeing* that someone has to deal with conflicts.  If that person is you, I don't mind so much, but it's likely that you're creating work for someone else by doing this.  

If you want to raise a separate PR that just reformats code, that's fine.

Reply:
Right, that's one of the more elaborate spellings if "Yes". (You're preaching to the choir - we all agree on all principles, but in an inconsistent codebase without checks, this will keep happening.)